Tests/Filter/BaseFilterTest.php

```diff
@@ -16,15 +16,15 @@
 
 class FilterTest_Filter extends BaseFilter
 {
-    public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $value)
+    public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $value): void
```

Comment:
This declaration must be changed according to this one: https://github.com/sonata-project/SonataDatagridBundle/pull/78/files#diff-16c85a051e0bc57d5f6ef974097290efR33

otherwise test won't pass.